CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Add per-frame encoding time in video encoding CPU connection health policy
```

Comment:
I messed this up in a previous PR (David's commit added `Unreleased` section) but will fix it myself :).